docs/backends.md

```diff
@@ -88,10 +99,35 @@ Content-Length: 129
 {"id":"ca219243-0dd4-4e6e-94ad-54fbf3dd8b32","name":"my dell server","description":"some description","address":"192.168.100.4"}
 ```
 
+## View Backend Information
+For a SEV enabled backend, the backend information can be viewed using prism. This information is measured by a rust script found [here](https://github.com/ultravioletrs/cocos/blob/main/scripts/backend_info/src/main.rs). Once compiled and the binary is stored in `/build`, the backend information can be measured by prism as shown below.
+
+On the backend page, click the Backend Information button:
+![backend_page](img/backend_page.png)
+
+If the measurement binary is absent and no measurement is found, an empty measurement file is displayed:
+
+![empty_info](img/empty_backend_info.png)
+
+If the measurement is present or measurement is found in the db, the measurement file will be available for download.
+
+![backend_info](img/backend_info.png)
+
+The backend_info.json file is useful in cocos when performing attestation. The file is provided to [cocos-cli](https://docs.cocos.ultraviolet.rs/cli/#backend-info) which can add measurement data or host data. The file contains the SnpPolicy and RootOfTrust as shown:
```

Comment:
attested tls, we do not use it to perform attestation

Reply:
it can also be used to verify attestation report. Also link to cocos docs where relevant